packages/terra-clinical-site/src/examples/header/HeaderLongText.jsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-alert */
+import React from 'react';
+import Header from 'terra-clinical-header';
+
+const LongTextWithButtons = () => (
+  <div>
+    <br />
+    <Header
+      title="Header Header Header Header Header Header Header Header Header Header Header Header Header Header Header Header Header Header"
```

Comment:
For these long text examples would you be against using Lorem Ipsum text?

http://generator.lorem-ipsum.info/_latin